downstairs/src/lib.rs

```diff
@@ -126,6 +126,48 @@ impl IOop {
     }
 }
 
+/// Read response data, containing data from all blocks
+#[derive(Debug)]
+pub struct RawReadResponse {
```

Comment:
Good idea, done in c39c8c7812ad017729976494432dae4f7c5af90f

(this also involved removing `into_read_responses`, since that was `#[cfg(test)]` which doesn't work across crate boundaries, but removing special-purpose test functions is generally good, so I don't mind)

Reply:
c35ff40f03992efa07169fe13ddea34901597430 moves `RawWrite` back into `crucible_protocol` as well (I had previously moved it into the upstairs crate, but agree it makes more sense in `crucible_protocol`)